workitem/workitem_repository.go

```diff
@@ -166,10 +294,20 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, typeID string, fiel
 	if err != nil {
 		return nil, errors.NewBadParameterError("type", typeID)
 	}
+
+	// The order of workitems are spaced by a factor of 1000.
+	pos, err := r.LoadHighestOrder()
+	if err != nil {
+		return nil, errors.NewInternalError(err.Error())
+	}
+	pos = pos + orderValue
+	fields[SystemOrder] = pos
```

Comment:
It's ok in this position, but why is SystemOrder not handled here https://github.com/DhritiShikhar/almighty-core/blob/e5b7d804073753a4737e13515df8f41493c8d783/workitem/workitem_repository.go#L333 and here https://github.com/DhritiShikhar/almighty-core/blob/e5b7d804073753a4737e13515df8f41493c8d783/workitem/workitem_repository.go#L427

Won't that just store system.order as part of the Fields jsonb when someone PATCH and not show up in WIList?